django_extensions/management/commands/sync_s3.py

```diff
@@ -70,13 +70,17 @@
 
 from django_extensions.management.utils import signalcommand
 
-# Make sure boto is available
-try:
-    import boto
-    import boto.exception
-    HAS_BOTO = True
-except ImportError:
-    HAS_BOTO = False
+
+def has_boto():
```

Comment:
Is there a special reason for making this into a function ? personally I dislike any use of `global` when we can avoid it and it also makes the `import boto` non obvious.

Reply:
yes, there is a reason .. I was unable to mock boto which was not installed.